### PR TITLE
Work directory

### PR DESCRIPTION
**Pull request by TraineeMax**
I have been doing tabBar class

### CODE REVIEW

File: ToDoTasks/Controllers/LoginViewController.swift [hunk 1]
```diff
@@ -152,10 +152,41 @@ class LoginViewController: UIViewController, UITextFieldDelegate {
         forgotPasswordButton.addTarget(self, action: #selector(forgotPasswordAction), for: .touchUpInside)
         return forgotPasswordButton
     }()
+    
+//    lazy var tabBarViewController: UITabBarController = {
```
Comment (MaxLaurel) [inline ToDoTasks/Controllers/LoginViewController.swift:156]: ll lnljbivtr

File: ToDoTasks/Controllers/PopoverViewController.swift [hunk 2]
```diff
@@ -7,17 +7,15 @@
 
 import UIKit
 
-class PopoverViewController: UIViewController {
```
Comment (MaxLaurel) [inline ToDoTasks/Controllers/PopoverViewController.swift:10]: mkmkmkmkmkkmkm

Comment (TraineeMax) [reply ToDoTasks/Controllers/PopoverViewController.swift:10]: jnjnjnjnjnjnnjn

File: ToDoTasks/Controllers/LoginViewController.swift (hunk 1 above)
Comment (TraineeMax) [reply ToDoTasks/Controllers/LoginViewController.swift:156]: mkmkmkmkmkmk

File: ToDoTasks.xcodeproj/project.pbxproj
Comment (MaxLaurel) [inline ToDoTasks.xcodeproj/project.pbxproj:1]: watched this

**Review by MaxLaurel — COMMENTED** [review]
It is ok but I can't approve it

**Review by MaxLaurel — APPROVED** [review]
you can merge this

**Pull request merged**